### ISSUE TITLE
Inquiry About Future Support for Controller Inputs in Flet

### ISSUE THREAD

**Issue by carimatics** [flet-dev/flet#2112]
Hello Flet Team,

I am currently working on a project using the Flet framework and am interested in incorporating controller(joystick) input, specifically for devices like the Nintendo Switch controller. I have searched through the documentation and existing issues but couldn't find relevant information about this feature.

I wanted to inquire if there are any plans to support controller inputs in the Flet framework in the near future. This feature would be highly beneficial for developing more interactive and diverse applications, especially those requiring non-traditional input methods.

If this is something that's already being considered or in development, could you please share any relevant details or progress updates? Additionally, if it's not currently on the roadmap, I would like to express my interest in this feature and hope to see it in future updates.

Thank you for your time and the excellent work on Flet. Looking forward to your response.

Best regards,
carimatics

**Comment by ndonkoHenri** [comment]
What do you think of this pypackage : https://pypi.org/project/pyjoystick/ ?

**Comment by carimatics** [comment]
Thank you for your prompt response.
I was not aware of that package until now.
Having briefly looked through the documentation, I feel it might be possible to integrate it into the Flet app I am developing.
I would definitely like to give it a try.

Thanks.

**Comment by ndonkoHenri** [comment]
Nice!
Will be great we move this thread to a discussion, wdyt?

**Comment by carimatics** [comment]
I think moving this thread to a discussion is a very good idea.

In my opinion, whether the code using pyjoystick works correctly or not, it would be very welcome if Flet could natively support controller input handling(like [Keyboard shortcuts](https://flet.dev/docs/guides/python/keyboard-shortcuts)).
However, I cannot gauge whether this request would be useful for many others as well.
To ascertain that, I think moving this thread to a discussion is a very good idea.

**Comment by carimatics** [comment]
Following your advice, I tried combining Flet with pyjoystick in my app.
As a result, I successfully achieved the desired behavior.
However, a new problem has arisen: the app does not launch when created with PyInstaller (it closes without any error message), but I have made progress from the initial issue.
Therefore, I would like to close this issue for now.
I am very grateful for your precise advice.

Additionally, I will create a new discussion regarding support for controller inputs.
Thank you as well for your advice on this matter.

Thanks!
